src/sciline/task_graph.py

```diff
@@ -78,4 +76,11 @@ def visualize(
         """
         from .visualize import to_graphviz
 
-        return to_graphviz(self._graph, **kwargs)
+        class _GraphVizualization:
+            def __init__(self, graph):
+                self.graph = graph
+
+            def _repr_html_(self):
+                return self.graph.pipe(format='svg', encoding='utf8')
+
+        return _GraphVizualization(to_graphviz(self._graph, **kwargs))
```

Comment:
I was wondering: Should the `to_graphviz` be moved into `GraphVisualization.__init__` (or another method)? Then we could also support other visualizations in the future. However it is then unclear how to pass the `kwargs` (since they are graphviz-specific), so I am not sure.

Or maybe a `staticmethod` `GraphVisualization.from_graphviz`, replacing `to_graphviz`?